Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:18.04
+MAINTAINER = path-help@sanger.ac.uk
+
+# dependency versions -- COULD specify these at build time
+ARG VR_CODEBASE_VERSION=0.04
+ARG CONFIG_GENERAL_VERSION=2.52
+
+RUN apt-get update -qq -y
+RUN apt-get upgrade -qq -y
```

Comment:
The image becomes outdated quickly anyways and base images are updated regularly enough imho so I tend to not run the upgrade.
Maybe something to discuss as a team though.